test/t8_gtest_macros.hxx

```diff
@@ -31,6 +31,7 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 #define T8_GTEST_MACROS_HXX
 
 #define AllEclasses testing::Range (T8_ECLASS_ZERO, T8_ECLASS_COUNT)
+#define AllEclasses2D testing::Range (T8_ECLASS_QUAD, T8_ECLASS_HEX)
```

Comment:
Why do you not use `Values(v1, v2, ..., vN)` here as we discussed in MM?

See http://google.github.io/googletest/reference/testing.html#INSTANTIATE_TEST_SUITE_P for reference.